### DISCUSSION TITLE
Media titles for whitewater kayaking!

### DISCUSSION THREAD

**Discussion by dhajoglou** [electronstudio/jaylib#32] [Show and tell]
I've been on this task for a while and finally made something I'm excited about!  This was on a Raspberry Pi 3 B+ and I used a java spring app with the JayLib java wrapper.  The box on the left is a countdown clock (45 second countdown) that's getting the data from a pubsub connection to a Redis Cache and the paddler name is sent via a REST api.  This is the first iteration where I will eventually build out heats, results, and a full race timing suite for downriver sprints.

I'm very happy with RayLib and very happy with the JayLib wrapper!

Thanks so much for building that armf jar for me!!!

<img width="870" alt="Screen Shot 2022-03-21 at 5 23 06 PM" src="https://user-images.githubusercontent.com/32944638/159381581-03e9dace-1d62-4153-b83e-cba3c1ebd0d2.png">

**Comment by electronstudio** [comment]
Cool.
